changes/12420-handle-policies-with-invalid-queries-desktop-endpoint

```diff
@@ -0,0 +1,2 @@
+- If a policy was defined with an invalid query, the desktop endpoint should count that policy as a
+  failed policy.
```

Comment:
This is a question for product:

Do we want to consider a host as "failing" a policy if:
- the SQL query has a typo, or,
- the query failed to execute on the host because the table doesn't exist, or,
- the query is too expensive that was killed during its execution
?

This is more of "an admin user to fix" responsibility rather than something to fix by the user of the device.
But I'm all ears. Also, for policy automations (webhooks, JIRA, etc.) we ignore the policy queries "results" that failed to execute on the hosts. We only consider a policy as "failed" on a host if it "completed successfully" and returned *no* results.

@zhumo